File: Capítulo_7/Sets/Método values/values.js
// values(): devolve um array com todos os valores (elementos) que estão no conjunto.
class Set {
    constructor() {
        this.items = {};
    }

    // Método values
    value() {
        return Object.values(this.items);
    }

    // Outro modo
    valuesLegacy(){
        let values = [];
        for (let key in this.items) {
            if (this.items.hasOwnProperty(key)) {
                values.push(self.items[key]);
            }
        }   
        return values;
    }
}

File: Capítulo_7/Sets/Método values/Código em Python/values.py
class Set:
    def __init__(self):
        self.items = {}
    
    # values(): devolve um array com todos os valores (elementos) que estão no conjunto.
    def values_in(self):
        return list(self.items.values())
    
    def values_in2(self):
        values = []
        for key in self.items:
            if self.has(key):
                values.append(self.items[key])
        return values
